tools/run_tests.py

```diff
@@ -67,50 +67,72 @@ def __download_cli_to_sandbox(self):
         return cli_filepath
 
 
-    def __configure_cli(self, sandbox_env, dcos_url):
+    def __configure_cli(self, dcos_url):
         cmds = [
+            'which dcos',
             'dcos config set core.dcos_url "{}"'.format(dcos_url),
             'dcos config set core.reporting True',
             'dcos config set core.ssl_verify false',
             'dcos config set core.timeout 5',
             'dcos config show']
         for cmd in cmds:
-            subprocess.check_call(cmd.split(' '), env=self.__sandbox_env)
+            subprocess.check_call(cmd.split(' '))
 
 
     def setup_cli(self):
         try:
             self.__github_updater.update('pending', 'Setting up CLI')
-            self.__sandbox_env = self.__get_cli_sandbox()
+            self.__configure_cli_sandbox() # adds to os.environ
```

Comment:
Small thing, but if we add linting for py scripts in this repo in the future, PEP8 specifies that end-of-line comments begin two spaces after the last non-comment, non-whitespace character in the line.